### DISCUSSION TITLE
Filter in qTable Column with QSelect

### DISCUSSION THREAD

**Discussion by Code-Case** [quasarframework/quasar#9122] [General - Components / Directives / etc] [answered]
Hi,

is it possible to use a QSelect with options generated with values from a QTable column to filter the QTable?
I have an input filter to search in the table and I need also a select dropdown to filter for countries.

I tryed something like this:

https://codepen.io/someOne11/pen/poRXKzP

**Comment by metalsadman** [comment, marked as answer]
`option-value="country"`. You probably need to use emit-value and map-options props as well, and maybe `option-label="country"` too.

**Reply by Code-Case** [reply]
updated my code example, Ive tested that with the same result, I think its because I need to run 2 different filter on the qTable?

**Reply by metalsadman** [reply]
So the dropdown is the keyword?

Here modified my old codepen https://codepen.io/metalsadman/pen/abpgJMG.

**Comment by Code-Case** [comment]
Thats nearly it, thank you very much, but I need to combine the qselect with a qinput.
And one last thing I have is that I need duplicate names showing only once in the Qselect like in your codepen name Gingerbread for example.

I made a Codepen with what I mean, currently the search term is passed to qselect because of the same filter I think and can we avoid this?

https://codepen.io/someOne11/pen/poRXKzP

best regards

**Reply by metalsadman** [reply]
You can simply make a computed properties for the data that filters for no duplicate and used it as options, as for filter models, use different ones for input and the qselects, then do filtering logic in the filter method.

**Reply by Code-Case** [reply]
Ah okay, now I use two different methods to filter, but I cant get the computed propperty working, Im getting object Object in my select menu as first Item. 

https://codepen.io/someOne11/pen/RwKXNRJ

**Reply by metalsadman** [reply]
please read more about computed props and what not down at vue docs.
anyway check this quasar app extension https://quasar-qgrid.netlify.app/examples.
